### PR TITLE
[3d] mapscene: Fix some edge cases of near/far plane computation

### PR DESCRIPTION
**Pull request by ptitjano**
This needs https://github.com/qgis/QGIS/pull/54551 to be merged first.

### CODE REVIEW

File: src/3d/qgs3dmapscene.cpp [hunk 1]
```diff
@@ -404,19 +404,21 @@ bool Qgs3DMapScene::updateCameraNearFarPlanes()
   if ( fnear < 1 )
     fnear = 1;  // does not really make sense to use negative far plane (behind camera)
 
-  // when zooming in a lot, fnear can become smaller than ffar. This should not happen
-  if ( fnear > ffar )
-    std::swap( fnear, ffar );
-
+  // the update didn't work out... this can happen if the scene does not contain
+  // any Qgs3DMapSceneEntity. Use the scen extent to compute near and far planes
```
Comment (nyalldawson) [inline src/3d/qgs3dmapscene.cpp:408]:
```suggestion
  // any Qgs3DMapSceneEntity. Use the scene extent to compute near and far planes
```

Comment (ptitjano) [reply src/3d/qgs3dmapscene.cpp]: Fixed.

**Comment by ptitjano** [comment]
@nyalldawson Gentle ping.

**Comment by ptitjano** [comment]
@wonder-sk or @uclaros Could you take a look at it please?

**Review by wonder-sk — APPROVED** [review]

**Pull request merged**